.gitignore

```diff
@@ -298,3 +298,9 @@ quickstart/csharp-unity/[Ll]ogs/
 # Linux executables
 quickstart/cpp-linux/helloworld
 samples/cpp/windows/console/samples/sample
+*.dll
+*.exe
+samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/objd/amd64/speechtotext-wpf.exe.config
+samples/csharp/dotnet-windows/speechtotext-wpf/speechtotext-wpf/objd/amd64/whatstheweatherlike.wav
+samples/batch/csharp/App.config
```

Comment:
that file is part of the project? why do you want to exclude it?

Reply:
no, accident. well spotted